src/onegov/event/collections/occurrences.py

```diff
@@ -272,15 +277,17 @@ def available_filters(self, sort_choices=False, sortfunc=None):
         :rtype tuple(tuples(keyword, title, values as list)
 
         """
+        assert self.event_filter_configuration
+        assert self.event_filter_fields
```

Comment:
I would just return an empty dict if they're not set. I don't think we need to require them to be set.

Reply:
allright